cmd/cluster-operator-controller-manager/app/controllermanager.go

```diff
@@ -536,6 +539,21 @@ func startClusterController(ctx ControllerContext) (bool, error) {
 	return true, nil
 }
 
+func startMachineAPIController(ctx ControllerContext) (bool, error) {
+	if !resourcesAvailable(ctx) {
```

Comment:
This always returns true since it is not checking for any resources.

Reply:
Updating to check for both API resources being ready as this uses both.